modules/Bio/EnsEMBL/Compara/PipeConfig/Parts/PrepareMasterDatabaseForRelease.pm

```diff
@@ -42,11 +42,11 @@ sub pipeline_analyses_prep_master_db_for_release {
     my ($self) = @_;
     return [
         {   -logic_name => 'patch_master_db',
-            -module     => 'Bio::EnsEMBL::Compara::RunnableDB::PrepareMaster::PatchMasterDB',
+            -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
             -parameters => {
-                'schema_file'  => $self->o('schema_file'),
-                'patch_dir'    => $self->o('patch_dir'),
-                'patch_names'  => '#patch_dir#/patch_' . $self->o('prev_release') . '_#release#_*.sql',
+                'patch_db_exe' => $self->o('patch_db_exe'),
+                'reg_conf'     => $self->o('reg_conf'),
+                'cmd'          => 'perl #patch_db_exe# --reg_conf #reg_conf# --reg_alias #master_db# --fixlast --nointeractive',
```

Comment:
The `perl` part in `cmd` can go.
The following is a suggestion so you don't need the additional parameter:
```suggestion
                'program' => $self->o('patch_db_exe'),
                'cmd'     => ['#program#', '--reg_conf', $self->o('reg_conf'), '--reg_alias', '#master_db#', '--fixlast', '--nointeractive'],
```
Nice addition of those two flags :+1:

Reply:
For the sake of consistency, I suggest to either keep `#reg_conf#` in `cmd` or to replace `#program#` with `$self->o('patch_db_exe')` (in `cmd`). What do you say?

Reply:
I just copied an example from an existing pipeline but your suggestion makes sense :+1: (I like more the second one, but up to you).

Reply:
Done :)